benchmark/benchmark_test.cpp

```diff
@@ -23,13 +23,64 @@
 
 using namespace ultrahdr;
 
-const std::string kDecTestVectors[] = {
-    "./data/city_night.jpg",    "./data/desert_sunset.jpg", "./data/lamps.jpg",
-    "./data/mountains.jpg",     "./data/desert_wanda.jpg",  "./data/grand_canyon.jpg",
-    "./data/mountain_lake.jpg",
+std::vector<std::string> kDecodeAPITestImages{
+    // 12mp test vectors
+    "mountains.jpg",
+    "mountain_lake.jpg",
+    "desert_wanda.jpg",
```

Comment:
Are these images checked in?

Reply:
these aren't checked in yet. Currently these will need to be manually placed. Once we get a confirmation, we will upload a zip that will contain these files.

Reply:
These images are shareable.

Reply:
Hi Harish, could you confirm if the images are going to be checked in together with this CL?

Reply:
@DichenZhang1 Images are uploaded to google cloud and the third commit in this pull request downloads the zip and uses those files.